src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -175,6 +175,7 @@ public Patch? CurrentPatch
         {
             currentPatch = value;
             playerHere.Visible = CurrentPatch == SelectedPatch;
+            ChemicalEquation.AutoRefreshProcess = true;
```

Comment:
This looks like it just sets the auto refresh to be on when this value is written. Instead I think what needs to be done is triggering the refresh when editor selected patch or current patch has changed.